From: Simon McVittie <smcv@debian.org>
Date: Thu, 5 Jul 2018 23:29:32 +0100
Subject: Remove unnecessary GObject cast

In recent GLib versions, g_object_ref() returns the type of its
parameter (previously it always returned a generic void *). In this
case, menuitem is a GtkWidget *, and priv->sec_activate_target wants
a GtkWidget *, but the G_OBJECT cast meant that g_object_ref() returned
a GObject *.

Signed-off-by: Simon McVittie <smcv@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=897473
---
 src/app-indicator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app-indicator.c b/src/app-indicator.c
index 6c86674..289601c 100644
--- a/src/app-indicator.c
+++ b/src/app-indicator.c
@@ -2185,7 +2185,7 @@ app_indicator_set_secondary_activate_target (AppIndicator *self, GtkWidget *menu
 
 	g_return_if_fail (GTK_IS_WIDGET (menuitem));
 
-	priv->sec_activate_target = g_object_ref(G_OBJECT(menuitem));
+	priv->sec_activate_target = g_object_ref(menuitem);
 	priv->sec_activate_enabled = widget_is_menu_child(self, menuitem);
 	g_signal_connect(menuitem, "parent-set", G_CALLBACK(sec_activate_target_parent_changed), self);
 }
